package.json

```diff
@@ -20,5 +21,4 @@
   "scripts": {
     "test": "mocha --require should --reporter spec"
   },
```

Comment:
I think technically this trailing comma is not allowed in pure JSON? (though iirc node.js and/or npm themselves do allow some funny business in these files…)

To roll a release I'll need to pull this repo back up locally and tweak the version anyway, so it's probably easier for me to merge this PR as is though and just fix it in a follow-on commit.

Reply:
done via 8e7ead94b1cf943a6461b36f6877ec30a482ce42